sources/models/src/lib.rs

```diff
@@ -207,6 +207,8 @@ struct KubernetesSettings {
     image_gc_low_threshold_percent: ImageGCLowThresholdPercent,
     provider_id: Url,
     log_level: u8,
+    tls_certificate: ValidBase64,
+    tls_private_key: ValidBase64,
```

Comment:
This should be `server_certificate` and `server_key` correct?

Reply:
Yep, I missed that rename. Fixed in update. Also rebased and addressed merged conflicts.